### PR TITLE
Add Compression bug to the list of known bugs

### PR DESCRIPTION
**Pull request by adamsitnik**
cc @jeffhandley @Jozkee

### CODE REVIEW

**Review by jeffhandley — APPROVED** [review]

**Comment by jeffhandley** [comment]
Let's actually hold off on merging this until we get the backport PR approved by Tactics. I'd like to add a statement about when it will be fixed.

File: release-notes/7.0/known-issues.md
Comment (jeffhandley) [inline release-notes/7.0/known-issues.md:9]:
```suggestion
It's impossible to decompress large (4GB+ after compression) zip archives that consist of more than one zipped file. There is no known workaround. More information and workaround can be found at: https://github.com/dotnet/runtime/issues/77159. This issue will be fixed in the December servicing release.
```

**Review by jeffhandley — CHANGES_REQUESTED** [review]

**Review by Eryk1705 — APPROVED** [review]

**Pull request merged**
